lib/puppetserver/ca/sign_action.rb

```diff
@@ -0,0 +1,174 @@
+require 'optparse'
+require 'openssl'
+require 'net/https'
+require 'json'
+
+module Puppetserver
+  module Ca
+    class SignAction
+
+      SUMMARY = "Set up the CA's key, certs, and crls"
+      BANNER = <<-BANNER
+Usage:
+  puppetserver ca sign [--help|--version]
+  puppetserver ca sign [--all] HOSTNAME
```

Comment:
I don't think this is exactly right. How do I specify that ``HOSTNAME`` is optional (ie, we want it if there's no ``--all`` flag, but we don't want it if there's an ``--all`` flag)?

Reply:
I think it might be more consistent with other commands in this gem to have the cert(s) be specified behind a flag, e.g. `puppetserver ca sign --certs certname1,certname2`.

Alternatively, the user can do `puppetserver ca sign --all`.